source.pem

```diff
@@ -0,0 +1,28 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
__Caution__: _DO NOT_ commit keys on GitHub. Remove this from here, and make sure you invalidate this public-private key pair as soon as possible.

Reply:
@notlmn how to do that??

Reply:
I suggest you use [GitHub Desktop](https://desktop.github.com) or similar to commit only the files and lines that are necessary — and then look at the PR and see if there's any unwanted code. We shouldn't have to comment on this. This file is unrelated and its changes should be undone